common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain time commitments"),
+                        "long_description": _(
+                            "State the time commitments for the course clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain course deadlines"),
+                        "long_description": _(
+                            "State the course deadlines and due dates (including time zones)"
+                            "clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain grading policy"),
+                        "long_description": _(
+                            "State the course grading policy and certificate requirements clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain learning objectives"),
+                        "long_description": _(
+                            "State the learning objectives, goals, and course outcome(s) clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain academic policy"),
+                        "long_description": _(
+                            "State the academic policy for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain forum participation expectations"),
+                        "long_description": _(
+                            "State the expectation for learner forum participation for the course on the course syllabus"
```

Comment:
Describe how learners participate in course discussions, and how participation affects the course experience.

question: I'm not sure how/if any course grade participation. What exactly is the goal that we're trying to achieve by asking them to include forum participation in the syllabus?